app/models/UserAnswers.scala

```diff
@@ -121,15 +129,20 @@ final case class UserAnswers(internalId: String,
 
 object UserAnswers {
 
-  def startNewSession(internalId: String, identifier: String, is5mldEnabled: Boolean, isTaxable: Boolean): UserAnswers =
-    UserAnswers(internalId = internalId, identifier = identifier, is5mldEnabled = is5mldEnabled, isTrustTaxable = isTaxable)
+  def startNewSession(internalId: String, identifier: String, is5mldEnabled: Boolean, isUnderlyingData5mld: Boolean): UserAnswers =
+    UserAnswers(
+      internalId = internalId,
+      identifier = identifier,
+      is5mldEnabled = is5mldEnabled,
+      isUnderlyingData5mld = isUnderlyingData5mld
+    )
 
   implicit lazy val reads: Reads[UserAnswers] = (
     (__ \ "internalId").read[String] and
       (__ \ "identifier").read[String] and
       (__ \ "data").read[JsObject] and
       (__ \ "is5mldEnabled").readWithDefault[Boolean](false) and
-      (__ \ "isTrustTaxable").readWithDefault[Boolean](true) and
+      (__ \ "isUnderlyingData5mld").readWithDefault[Boolean](false) and
```

Comment:
Is this rename going to be a breaking change in production? Although we are doing a defaults.